package.json

```diff
@@ -1,7 +1,4 @@
 {
-  "name": "tldr",
-  "version": "1.0.0",
-  "description": "Simplified, community-driven man pages",
```

Comment:
Not sure that these should be removed. Perhaps we could update them to be different to the tldr-node-client?

I believe removing these will cause npm to get upset and warn you.

Reply:
It won't, since neither is required. I tested this on my own PC and it seems no warning are given. :open_mouth:

Reply:
Ah, that must have changed then. I'm still uneasy about removing them though.

Reply:
I think as long as `private` is set to `true`, then these won't flag an error. They're only required for publishing to a package repository afaik.

Reply:
True @pxgamer, but I think the metadata here is valuable to keep - even if we rewrite it.